### ISSUE TITLE
Multiple query parameters in rails link helpers will cause a RuntimeError: Illegal character '&'

### ISSUE THREAD

**Issue by treznick** [zurb/inky-rb#15]
Using rails link helpers with multiple params will put `&` characters in the generated urls. When inky-rb tries to parse this, it blows up.

**Comment by treznick** [comment]
As a workaround, you can XML escape the `&` in the generated url. Here's one way:
``` 
<%= Builder::XChar.encode(link_to(@my_model, first_param: "foo", second_param: "bar")) %>
```

**Comment by kball** [comment]
Bizarre!  REXML is weirder and weirder the more I use it.

This is fixed by https://github.com/zurb/inky-rb/pull/19 by @francirp so once we get that PR finalized this will be fixed.

**Comment by kball** [comment]
Fixed by #19
